tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -18,36 +18,111 @@ This document describes how to create a changefeed to stream data from TiDB Clou
 - Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#unsupported-scenarios).
 - If the table to be replicated does not have a primary key or a non-null unique index, the absence of a unique constraint during replication could result in duplicated data being inserted downstream in some retry scenarios.
 
-## Create a changefeed
+## Step 1. Create a changefeed
```

Comment:
I broke the long process down into three major steps. They'll be easier to follow.